### ISSUE TITLE
Incorrect serialization of collections in request classes

### ISSUE THREAD

**Issue by kobylynskyi** [kobylynskyi/graphql-java-codegen#138]
## Issue Description

If there is an input class with a collection of strings, then serialization to GraphQL request is wrong.

Sample schema:
```graphqls
mutation {
    updateTags(tags: [TagInput!]!): Tag
}

input TagInput {
    key: String!
    values: [String]
}
```
Sample code:
```java
new TagInput.Builder()
    .setKey("smthng")
    .setValues(Arrays.asList("1", "2", "3"))
    .build();
```

## Actual GraphQL request
```graphqls
mutation { updateTags(tags: [{key: "smthng", values: [1, 2, 3]}]) }
```

## Expected GraphQL request
```graphqls
mutation { updateTags(tags: [{key: "smthng", values: ["1", "2", "3"]}]) }
```

**Comment by kobylynskyi** [comment]
This should be fixed and released early next week.
CC @JustinBlandford (not sure you faced this issue before)

**Comment by kobylynskyi** [comment]
The fix was released as part of 1.7.3
